components/WorkExp.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import WorkExpCard from "./WorkExpCard";
+import Image from "next/image";
+
+type Props = {};
+
+const WorkExp = (props: Props) => {
```

Comment:
Are you planning on passing props into `WorkExp`?

Reply:
No. That's there because I ran tsrafce and forgot to remove it.